creator-node/src/TranscodingQueue.js

```diff
@@ -78,16 +79,20 @@ class TranscodingQueue {
             logContext
           )
 
-          const filePath = await ffmpeg.transcodeFileTo320(fileDir, fileName, {
-            logContext
-          })
+          const transcodeFilePath = await ffmpeg.transcodeFileTo320(
+            fileDir,
+            fileName,
+            {
+              logContext
+            }
+          )
           this.logStatus(
             `Successfully completed Transcode320 job ${fileDir} ${fileName} in duration ${
               Date.now() - start
             }ms`,
             logContext
           )
-          done(null, { filePath })
+          done(null, { transcodeFilePath })
```

Comment:
could you update the documentation for segment and transcode320 below to show the properties of the return object